common/pc/display/hidpi.nix

```
@@ -0,0 +1,17 @@
+{ pkgs, lib, ... }: {
+  console.font = lib.mkDefault
+    "${pkgs.terminus_font}/share/consolefonts/ter-u28n.psf.gz";
+
+  # Needed when doing cryptsetup
+  console.earlySetup = lib.mkDefault true;
+  boot.loader.systemd-boot.consoleMode = lib.mkDefault "max";
+
+  services.xserver.dpi = lib.mkDefault 196;
+  fonts.fontconfig.dpi = lib.mkDefault 196;
```

This is what I currently:

```suggestion
  services.xserver.displayManager.setupCommands = ''
    ${pkgs.xlibs.xrandr}/bin/xrandr --dpi eDP-1
  '';
```

It simply sets the dpi of my 4k screen as the dpi of x11. Not sure how to make the `eDP-1` part more general.